docs/src/appendix/cast/multicall.md

```diff
@@ -27,4 +34,18 @@ call_type = "invoke"
 contract_address = "map_contract"
 function = "put"
 inputs = ["0x123", "234"]
-```
\ No newline at end of file
+```
+
+## `new`
+
+Generates an empty template for the multicall `.toml` file that may be later use with the `run` subcommand.
```

Comment:
let's add an info, that by default it is being printed to stdout here

Reply:
[address dox fdbck](https://github.com/foundry-rs/starknet-foundry/pull/98/commits/722855d7bb207da35ea14576b497377f59274d74)